dash_labs/plugins/pages.py

```diff
@@ -224,16 +243,27 @@ def _filename_to_name(filename):
     return filename.split(".")[-1].replace("_", " ").capitalize()
 
 
-def _filename_to_path(filename):
-    return filename.replace("_", "-").replace(".", "/").lower().split("pages")[-1]
+def _validate_template(template):
+    template_segments = template.split("/")
+    for s in template_segments:
+        if ("<" or ">") in s and not (s.startswith("<") and s.endswith(">")):
```

Comment:
`("<" or ">") in s`  -> I think you mean `("<" in s or ">" in s)` 

Also need to check that `<` and `>` are NOT in `s[1: -1]`

Reply:
> Also need to check that < and > are NOT in s[1: -1]

Just to summarize our discussion:  Since the variables are passed to the layout in a dict as strings, there should not be a restriction about including  `<` or ` >` .   However, the most common use case is that the keys are Python variables, so now there is a warning message if the variable is not a valid Python variable name.